### ISSUE TITLE
Improve "Edit and Repeat" feature

### ISSUE THREAD

**Issue by kunbr0** [ProxymanApp/Proxyman#1353]
## Description
Improve "Edit and Repeat" feature

## Expected Behavior
The input area can be expand to multiple lines (like <textarea> in HTML) so that user can edit params in GET requests easier.

## Current Behavior
The input area of URL now is inline (one line).
![image](https://user-images.githubusercontent.com/55029544/188843420-900999c6-ca91-4628-80b6-df5984c2db92.png)

## Steps to Reproduce
1. Right click to any captured request
2. Select "Edit and Repeat..."

## Environment
- App version: e.g Proxyman 3.9.0
- macOS version: 12.4

**Comment by NghiaTranUIT** [comment]
Thanks for your suggestion @kunbr0, but it's hard to implement the multiple lines since the NSTextField is inside the [NSToolBar](https://developer.apple.com/documentation/appkit/nstoolbar), which doesn't allow it to expand its height. 

Alternatively, You can directly edit the query string from the Raw Tab.

<img width="1223" alt="Screen_Shot_2022-09-08_at_08_26_14" src="https://user-images.githubusercontent.com/5878421/189013351-299698b4-393b-4e7a-ac37-121989e45f43.png">

**Comment by toantan-me** [comment]
Oh. Thank you.
Also, hope that you can add more feature (like MapLocal, Scripting,..) for ProxyMan on IOS soon.
Very great product. 😄

**Comment by kunbr0** [comment]
Oh. Thank you.
Also, hope that you can add more feature (like MapLocal, Scripting,..) for ProxyMan on IOS soon.
Very great product. 😄

**Comment by NghiaTranUIT** [comment]
Yes, soon. Our team is going to start working on the iOS version 👍
